### ISSUE TITLE
How could I convert a tensorflow model which include Batch Normalization using tfcoreml, since the converter requires to insert the Boolean shape of the batch normalization

### ISSUE THREAD

**Issue by tsgtdss583** [tf-coreml/tf-coreml#280]
convOutput = tf.layers.batch_normalization(convInput, training=training_phase2)
the 'training' argument is a placeholder to receive the boolean input. how could I set the shape of a boolean while converting a model to coreML

**Comment by danvargg** [comment]
Facing the exact same issue here.

**Comment by aseemw** [comment]
Did you try setting the shape as (1,) ?

**Comment by danvargg** [comment]
Thanks. I will give it a try and get back to you.

**Comment by danvargg** [comment]
Hi, it worked! Thanks a lot!.

But I wonder, why are `switch` operations created if I save and freeze the model with `is_training = False`. Shouldn't the `is_training = True` path be absent?
